packages/conf-m4/conf-m4.1/opam

```diff
@@ -4,7 +4,7 @@ homepage: "http://www.gnu.org/software/m4/m4.html"
 bug-reports: "https://github.com/ocaml/opam-repository/issues"
 dev-repo: "https://github.com/ocaml/opam-repository.git"
 license: "GPL-3"
-build: [["m4" "--version"]]
+build: [["sh" "-c" "echo | m4"]]
```

Comment:
I think this needs `-e` because, in non-interactive shells, the error may be thrown away and `-x` might be nice, too.